src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/citizen/RespondToClaimTest.java

```diff
@@ -94,7 +128,7 @@ private void shouldBeAbleToSuccessfullySubmit(Response response) {
 
         assertThat(updatedCase.getResponse().isPresent()).isTrue();
         assertThat(updatedCase.getResponse().get()).isEqualTo(response);
-        assertThat(updatedCase.getRespondedAt()).isCloseTo(nowInLocalZone(), within(2, ChronoUnit.MINUTES));
+        assertThat(updatedCase.getRespondedAt()).isCloseTo(nowInUTC(), within(2, ChronoUnit.MINUTES));
```

Comment:
can't we just get rid of these isCloseTo assertions - they are bollocks 😄 or just check getRespondedAt is not null don't compare to another time